binding_generator.py

```diff
@@ -1723,6 +1723,12 @@ def generate_engine_class_header(class_api, used_classes, fully_used_classes, us
         result.append(f"\t~{class_name}();")
         result.append("")
 
+    if class_name == "Object":
+        result.append(
+            '\tString _to_string() const { return "[" + String(get_class()) + ":" + itos(get_instance_id()) + "]"; }'
```

Comment:
Current master uses `"<" + get_class() + "#" + itos(get_instance_id()) + ">"`. Should we match?

Reply:
What about Wrapped? Should that get the pointy brackets too?

Reply:
Oh, right, thanks! Fixed in my latest push, but that one should hopefully never be called :-)